### PR TITLE
Subscriptions can now be set to false (usefull for federation gateway) + readme update

### PR DESCRIPTION
**Pull request by chaptiste**
Subscriptions need to be disabled when using apollo gateway, and this wasn't possible (as far as I know), so I made this very simple fix.

### CODE REVIEW

**Comment by chaptiste** [comment]
I also edited the readme as graphql 16 (the latest) isn't compatible with the apollo server version used in this package.

**Comment by sebastiandittrich** [comment]
Thank you very much for this PR! I really appreciate your effort to make this a better package. Unfortunately, I have a lot to do currently so you can expect me to review this in one or two weeks at the earliest.

With kind regards
Sebastian Dittrich

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is an Apollo Server integration for Adonisjs 5.
 ## Quickstart
 
 ```sh
-npm install graphql apollo graphql-tag apollo-adonis
+npm install graphql@15.8.0 apollo graphql-tag apollo-adonis
```
Comment (sebastiandittrich) [inline README.md]: Do we need to pin this dependecy to this very specific version? Or is it enough to use graphql@15? Or maybe graphql@15.8? Pinning it to 15.8.0 will disable any updates (also security related bug fixes) for this package in the future.

**Review by sebastiandittrich — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chaptiste) [reply README.md]: Yes my bad, graphql@15.8 is better, there's no need to have this specific version. As long as it's not graphql@16 it should be good !

**Comment by sebastiandittrich** [comment]
Your changes are now merged and included in version v1.1.0. Thank you very much for your contribution! 😄  (And sorry that it took so long.)

**Pull request merged**